lib/paper_trail.ex

```diff
@@ -322,7 +322,7 @@ defmodule PaperTrail do
     %Version{
       event: "update",
       item_type: changeset.data.__struct__ |> Module.split() |> List.last(),
-      item_id: changeset.data.id,
+      item_id: get_model_id_from_changeset(changeset),
```

Comment:
This could simply be get_model_id(changeset.data) instead of a new function I think